modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/info/CSDiagramEntryInfoItemFields.java

```diff
@@ -0,0 +1,156 @@
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.product.content.web.internal.info;
+
+import com.liferay.commerce.shop.by.diagram.model.CSDiagramEntry;
+import com.liferay.info.field.InfoField;
+import com.liferay.info.field.type.BooleanInfoFieldType;
+import com.liferay.info.field.type.DateInfoFieldType;
+import com.liferay.info.field.type.NumberInfoFieldType;
+import com.liferay.info.field.type.TextInfoFieldType;
+import com.liferay.info.localized.InfoLocalizedValue;
+
+/**
+ * @author Mahmoud Azzam
+ */
+public class CSDiagramEntryInfoItemFields {
+
+	public static final InfoField<NumberInfoFieldType> companyIdInfoField =
+		CSDiagramEntryInfoItemFields.BuilderHolder._builder.infoFieldType(
+			NumberInfoFieldType.INSTANCE
+		).name(
+			"companyId"
+		).labelInfoLocalizedValue(
+			InfoLocalizedValue.localize(
+				CSDiagramEntryInfoItemFields.class, "company-id")
+		).build();
+	public static final InfoField<NumberInfoFieldType> cpDefinitionIdInfoField =
+		CSDiagramEntryInfoItemFields.BuilderHolder._builder.infoFieldType(
+			NumberInfoFieldType.INSTANCE
+		).name(
+			"cpDefinitionId"
+		).labelInfoLocalizedValue(
+			InfoLocalizedValue.localize(
+				CSDiagramEntryInfoItemFields.class, "cp-definition-id")
+		).build();
+	public static final InfoField<NumberInfoFieldType> cpInstanceIdInfoField =
+		CSDiagramEntryInfoItemFields.BuilderHolder._builder.infoFieldType(
+			NumberInfoFieldType.INSTANCE
+		).name(
+			"cpInstanceId"
+		).labelInfoLocalizedValue(
+			InfoLocalizedValue.localize(
+				CSDiagramEntryInfoItemFields.class, "cp-instance-id")
+		).build();
+	public static final InfoField<NumberInfoFieldType> cProductIdInfoField =
+		CSDiagramEntryInfoItemFields.BuilderHolder._builder.infoFieldType(
+			NumberInfoFieldType.INSTANCE
+		).name(
+			"cProductId"
+		).labelInfoLocalizedValue(
+			InfoLocalizedValue.localize(
+				CSDiagramEntryInfoItemFields.class, "c-product-id")
+		).build();
+	public static final InfoField<DateInfoFieldType> createDateInfoField =
+		CSDiagramEntryInfoItemFields.BuilderHolder._builder.infoFieldType(
+			DateInfoFieldType.INSTANCE
+		).name(
+			"createDate"
+		).labelInfoLocalizedValue(
+			InfoLocalizedValue.localize(
+				CSDiagramEntryInfoItemFields.class, "create-date")
+		).build();
+	public static final InfoField<NumberInfoFieldType>
+		csDiagramEntryIdInfoField =
+			CSDiagramEntryInfoItemFields.BuilderHolder._builder.infoFieldType(
+				NumberInfoFieldType.INSTANCE
+			).name(
+				"csDiagramEntryId"
+			).labelInfoLocalizedValue(
+				InfoLocalizedValue.localize(
+					CSDiagramEntryInfoItemFields.class, "cs-diagram-entry-id")
```

Comment:
csDiagramEntryId

Reply:
do you want me to change cs-diagram-entry-id to csDiagramEntryId in Language.properties and CSDiagramEntryInfoItemFields.java?

Reply:
No if no other translation were added for other ids, I think that they were not needed, so I don't think you need to add a translation here.

Reply:
```suggestion
					CSDiagramEntryInfoItemFields.class, "csDiagramEntryId")
```